bin/casher

```diff
@@ -90,7 +90,7 @@ function fetch {
 
     msg "fetching ${display_name}" green
 
-    curl $url -o ${FETCH_TAR} -f -s --retry 3 >fetch.log 2>fetch.err.log
+    curl $url -o ${FETCH_TAR} -f -s --retry 3 >${CASHER_DIR}/fetch.log 2>${CASHER_DIR}/fetch.err.log
```

Comment:
Is there a reason that variables aren't within `""`?

Reply:
I went for consistency with another log writer in the same file which also does not use quotes. CASHER_DIR is likely fixed to a path with no unsafe characters, so in this occasion I can get away with not quoting.

This whole file could use some defensive programming (quoting), but that is beyond the scope of this PR.